docs/design/rules-api.md

```diff
@@ -0,0 +1,119 @@
+# Rules API
+
+Observatorium is featured with a multi-tenant API that enables tenants to write and read Prometheus recording and alerting rules.
+
+The API is [defined](https://github.com/observatorium/api/blob/main/rules/spec.yaml) using the [OpenAPI](https://swagger.io/specification/) specification format.
+
+The goal is to enable tenants to create, modify and access their own rules.
+
+## Usage
+
+### Create rules
+
+```
+PUT /api/v1/rules/raw
+```
+
+Set rules for a tenant.
+
+#### Example request
+
+```bash
+curl -X PUT --data-binary @alerting-rule.yaml --header "Content-Type: application/yaml" http://<observatorium-api-url>/api/metrics/v1/<tenant>/api/v1/rules/raw
```

Comment:
We should also mention something about authentication headers here.

Reply:
I guess a common note saying that all these requests are tenant scoped and requires a valid auth token in the `Authorization` header would do the trick.

Reply:
good idea, added!